cmd/annotations_prep/main.go

```diff
@@ -147,32 +148,79 @@ Istio supports to control its behavior.
 `
 )
 
+// Collection represents template fields for either annotations or labels.
+type Collection struct {
+	Name                string
+	NamePlural          string
+	NameLowercase       string
+	NameLowercasePlural string
+	// Link is the location of the generated page on istio.io.
+	Link string
+	// ConceptLink is the link to the concept page for the collection type.
+	ConceptLink string
+}
+
+var (
+	annotations = Collection{
+		Name:                "Annotation",
+		NamePlural:          "Annotations",
+		NameLowercase:       "annotation",
+		NameLowercasePlural: "annotations",
+		Link:                "https://istio.io/latest/docs/reference/config/annotations/",
+		ConceptLink:         "https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/",
+	}
+
+	labels = Collection{
+		Name:                "Label",
+		NamePlural:          "Labels",
+		NameLowercase:       "label",
+		NameLowercasePlural: "labels",
+		Link:                "https://istio.io/docs/reference/config/labels/",
```

Comment:
This is a dead link.

Reply:
It's a chicken-or-the egg issue. It can't be live until I use this tool to publish the labels html.